src/resources/Activities/ActivitiesInterfaces.ts

```diff
@@ -73,3 +73,10 @@ export interface ActivityFacetModel {
     sections?: string[];
     states?: string[];
 }
+
+export interface ActivitiesResourceAndOperations {
+    /**
+    * Will contain resources and their corresponding operations, where the key is the name of the resource.
```

Comment:
This is a nitpick, but unless this comes from the API docs, I prefer to use the present tense for descriptions of a return type.
```suggestion
    * Contains resources and their corresponding operations, where the key is the name of the resource.
```

Reply:
All good thanks